pkg/capture/flow.go

```diff
@@ -215,9 +221,17 @@ func (f *FlowLog) Aggregate() (agg *hashmap.AggFlowMap) {
 
 	agg = hashmap.NewAggFlowMap()
 
+	// for recomputing the most up to date running sum of bytes and packets
+	f.totals.Reset()
+
 	// Reusable key conversion buffers
 	keyBufV4, keyBufV6 := types.NewEmptyV4Key(), types.NewEmptyV6Key()
 	for _, v := range f.flowMap {
+		// update totals
+		f.totals.BytesRcvd += v.bytesRcvd
```

Comment:
Please move inside the condition below (no need to count if zero packets are observed)